intermediate_source/rpc_param_server_tutorial.rst

```diff
@@ -0,0 +1,382 @@
+
+.. code-block:: python
+
+   processes = []
+   world_size = args.world_size
+   if args.rank == 0:
+       p = mp.Process(target=run_parameter_server, args=(0, world_size))
+       p.start()
+       processes.append(p)
+   else:
+       # Get data to train on
+       train_loader = torch.utils.data.DataLoader(
+           datasets.MNIST('../data', train=True, download=True,
+                          transform=transforms.Compose([
+                              transforms.ToTensor(),
+                              transforms.Normalize((0.1307,), (0.3081,))
+                          ])),
+           batch_size=32, shuffle=True,)
+       test_loader = torch.utils.data.DataLoader(
+           datasets.MNIST(
+               '../data',
+               train=False,
+               transform=transforms.Compose(
+                   [
+                       transforms.ToTensor(),
+                       transforms.Normalize(
+                           (0.1307,
+                            ),
+                           (0.3081,
+                            ))])),
+           batch_size=32,
+           shuffle=True,
+       )
+       # start training worker on this node
+       p = mp.Process(
+           target=run_worker,
+           args=(
+               args.rank,
+               world_size, args.num_gpus,
+               train_loader,
+               test_loader))
+       p.start()
+       processes.append(p)
+
+   for p in processes:
+       p.join()
+To run the example locally, run the following command worker for the server and each worker you wish to spawn, in separate terminal windows: ``python rpc_parameter_server.py --world_size=WORLD_SIZE --rank=RANK``. For example, for a master node with world size of 2, the command would be ``python rpc_parameter_server.py --world_size=2 --rank=0``. The trainer can then be launched with the command ``python rpc_parameter_server.py --world_size=2 --rank=1`` in a separate window, and this will begin training with one server and a single trainer. Note that this tutorial assumes that training occurs using between 0 and 2 GPUs, and this argument can be configured by passing ``--num_gpus=N`` into the training script.
+
+You can pass in the command line arguments ``--master_addr=<address>`` and ``master_port=PORT`` to indicate the address and port that the master worker is listening on, for example, to test functionality where trainers and master nodes run on different machines.
```

Comment:
master_port -> --master_port?

BTW, is the format difference between `<address>` and `PORT` intentional?